### PR TITLE
LPD-23558 Investigate failing test com.liferay.exportimport.internal.content.processor.test.DefaultExportImportContentProcessorTest

### PR DESCRIPTION
**Pull request by Mikellorza**
https://liferay.atlassian.net/browse/LPD-23558

Description 
-------
Do not use here the CANONICAL_URL_SEPARATOR to use in the validate layout reference method because if not the control panel urls will be correct, and it is invalid

### CODE REVIEW

File: modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/exportimport/content/processor/LayoutReferencesExportImportContentProcessor.java [hunk 1]
```diff
@@ -941,7 +943,21 @@ else if ((content.charAt(beginPos) == CharPool.OPEN_BRACKET) ||
...
+								s,
+								VirtualLayoutConstants.
+									CANONICAL_URL_SEPARATOR)) {
+
+							return null;
+						}
+
+						return s;
+					},
+					String.class));
```
Comment (adolfopa) [inline modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/exportimport/content/processor/LayoutReferencesExportImportContentProcessor.java:960]: Wouldn't it be simpler to user `List#remove`?
```
List<String> urlSeparators = FriendlyURLResolverRegistryUtil.getURLSeparators();

urlSeparators.remove(VirtualLayoutConstants.CANONICAL_URL_SEPARATOR);
```

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/exportimport/content/processor/LayoutReferencesExportImportContentProcessor.java (hunk 1 above)
Comment (Mikellorza) [reply modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/exportimport/content/processor/LayoutReferencesExportImportContentProcessor.java:960]: Yes, but getURLSeparators  returns String[] and we have to transform to a List<String>, so I think that it is more efficient to transform the array directly to another array without the CANONICAL_URL_SEPARATOR. What do you think?

Comment (adolfopa) [reply modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/exportimport/content/processor/LayoutReferencesExportImportContentProcessor.java:960]: You're right :man_facepalming:  I thought it was a list. Sorry!

**Review by adolfopa — APPROVED** [review]

File: modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/exportimport/content/processor/LayoutReferencesExportImportContentProcessor.java (hunk 1 above)
Comment (adolfopa) [reply modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/exportimport/content/processor/LayoutReferencesExportImportContentProcessor.java:960]: But what about `ArrayUtil#remove` :wink:

Comment (Mikellorza) [reply modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/exportimport/content/processor/LayoutReferencesExportImportContentProcessor.java:960]: Oh yes, let's wait a moment I am changing it right now.

thanks!

Comment (Mikellorza) [reply modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/exportimport/content/processor/LayoutReferencesExportImportContentProcessor.java]: I've just changed it, thanks @adolfopa

**Comment by Mikellorza** [comment]
ci:test:sf

**Comment by Mikellorza** [comment]
ci:forward

**Comment by 4lejandrito** [comment]
Gracias!!

**Pull request closed without merging**